### ISSUE TITLE
Corrections from Product Owner

### ISSUE THREAD

**Issue by pisipite** [themoonsheep/project-assetdeclarations-hungary#1]
**Questions**

- There was the plan for help bubbles, which would help matching the fields on the two sides. It might be easiert to put two words next to the name of the fields (basically the number or letter of the question) with smaller font size.  Would that work? Shall I write these notes?
- Is it possible to auto separate every three digits of the number (eg. 7 000 000)?

**Corrections**

- [x] move newsletter subscription field to the very end of the questions, just before the "Liberate/Felszabadítom" button
 - [ ] subscribe with clicking "Liberate/Felszabadítom" button (no separate button needed).
- [ ] insert a short text field between the K-Monitor logo and MP picture for volunteer nickname - for it is optional, but recommended for identification of the user - if provided save it in cookie
- [ ] no need to repeat the title "Gyermek:" when adding +1 child (only a new field should appear)
- [x] same (smaller) font size as before for legal designation, legal status and ownership ratio ("Ingatlan jogi jellege", "Nyilatkozó jogállása", Tulajdoni hányad)
- [ ] remove the title acquisition title ("Szerzés jogcíme") with bigger font, keep only the smaller
- [x] delete the property_category question ("Melyik típusba sorolnád az ingatlant?"). It was for Vagyonkereső.hu, but we don't really need it.
- [x] put the something_wrong box to the very end of the questions, just before the 
 newsletter subscribtion and "Liberate/Felszabadítom" button
- [ ] Title for the first field of "Védett műalkotás, védett gyűjtemény és egyéb nagyobb értékű ingóságok" is missing: Ingóság jellege.
- [x] Change "+1 műtárgy" to "+1 ingóság"
- [ ] We missed out one field: in the PDFs it's "II./9. Más, jelentősebb értékű..." It should be placed at the end of the "Nyilatkozik-e bármilyen megtakarításról, befektetésről, pénzkövetelésről?" (savings) section, after the "Pénzintézeti számlakövetelés vagy más pénzkövetelés." Title: "Más jelentős értékű vagyontárgy :"; [short text field]; "+1" button.
- [x] in-field text for benefit_name "megnevezés"
- [ ] drop down options for benefit_curr: "HUF"; "törvény szerint" (instead of "törvényben alapján"); "egyéb"
- [ ] drop down oprions for present_currency: "HUF"; "egyéb"

**Comment by KrzysztofMadejski** [comment]
> the number or letter of the question in the original pdf. eg 1. a) and 1. b) on this screensjot.
![capture](https://user-images.githubusercontent.com/849975/34034719-e871f254-e17f-11e7-9fa3-8f116f89db3b.PNG)

**Comment by KrzysztofMadejski** [comment]
- [ ] juttatás options: add `egyeb` to have `HUF; törvény alapján [according to the legislation]; egyéb [other].`

**Comment by pisipite** [comment]
data export to Vagyonkereső (by Direkt36) fields: https://drive.google.com/file/d/0B2uyEiZ5YeiAUlZkdjVLTlg0MHE0YmFaNmw3ZThZcW9JTTZN/view?usp=sharing

@KrzysztofMadejski: I've moved it to a separate issue: https://github.com/themoonsheep/project-assetdeclarations-hungary/issues/8

**Comment by KrzysztofMadejski** [comment]
I won't touch Direkt36 export unless they report back about the export I did one year ago..

**Comment by pisipite** [comment]
👍 It's definetly a priority to make our thing work.
I'll try to meet the guy this week.

**Comment by KrzysztofMadejski** [comment]
Once above is done, let's make sure the following correction requests are resolved (they were send against the prototype):
- [ ] **at the end of a property i can tick all the radio buttons (for categories) at the same time, but i can't untick them afterwords.**
- [ ] change cultivation from checkboxes to drop down similar to area_unit. three given option + pop up other box
- [ ] change building_nature from checkboxes to drop down similar to area_unit. three given option + pop up other box
- [ ] change legal_designation from checkboxes to drop down similar to area_unit. three given option + pop up other box
- [ ] change legal_status from checkboxes to drop down similar to area_unit. three given option + pop up other box
- [ ] move something_wrong to the very end of the form (just before the “liberate” button)
- [ ] at the end of savings, investments, cash section we need one more question. name: Egyéb értékes vagyontárgy (Other valuable asset); single line textbox with a +1 button

**Comment by KrzysztofMadejski** [comment]
Check if these has been resolved as well: https://github.com/the-engine-room/replication-sprint-02/issues/29
